### PR TITLE
2024 migration

### PR DESCRIPTION
**Pull request by Yourself1011**

### CODE REVIEW

**Comment by jeffrey-zang** [comment]
Also need to update CloudFlare env vars

File: .env.template [hunk 1]
```diff
@@ -1,5 +1,11 @@
...
-VITE_DISCORD_CLIENT_ID=
-VITE_BACKEND_URL=https://rythmbot.up.railway.app
\ No newline at end of file
+# Environment variables declared in this file are automatically made available to Prisma.
+# See the documentation for more detail: https://pris.ly/d/prisma-schema#accessing-environment-variables-from-the-schema
+
+# Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server, MongoDB and CockroachDB.
+# See the documentation for all the connection string options: https://pris.ly/d/connection-strings
+
+DATABASE_URL="postgresql://danielzhang@localhost:5432/rythmhacksdash"
```
Comment (jeffrey-zang) [inline .env.template:7]: erm is this prod ready

Comment (Yourself1011) [reply .env.template:7]: no

**Comment by jeffrey-zang** [comment]
a lot of the UI code in this repo is crap. we should redo most of it

**Comment by jeffrey-zang** [comment]
looks like it works. i'm too lazy to test prisma + postgres so 

LGTM

**Review by jeffrey-zang — APPROVED** [review]

**Comment by jeffrey-zang** [comment]
some of the env var imports are messed up, ie.  src/app/dashboard/discord/page.tsx

**Review by lifeng-yin — APPROVED** [review]
Looks good!

**Pull request merged**